<launch>

    <arg
            name="gait_package"
            default="march_gait_files"
            doc="The package where the gait files are located"
    />
    <arg
            name="gait_directory"
            default="training-v"
            doc="The directory where the gait files are located, relative to the above package"
    />
    <arg name="sounds" default="false" doc="Whether to use sounds."/>

    <node
            name="gait_selection_node"
            pkg="march_gait_selection"
            type="march_gait_selection_node"
            output="screen"
            required="true">
        <param name="gait_package" value="$(arg gait_package)"/>
        <param name="gait_directory" value="$(arg gait_directory)"/>
        <param name="sounds" value="$(arg sounds)"/>
    </node>
</launch>
